lib_test/dune

```diff
@@ -1,5 +1,11 @@
 (tests
  (libraries mustache ounit2 ezjsonm)
  (names test_mustache spec_mustache)
- (deps test_mustache.exe ../specs/comments.json ../specs/interpolation.json
-   ../specs/partials.json ../specs/sections.json ../specs/inverted.json))
+ (deps test_mustache.exe
+   ../specs/comments.json
+   ../specs/inheritance.json
+   ../specs/interpolation.json
+   ../specs/partials.json
+   ../specs/sections.json
+   ../specs/inverted.json
```

Comment:
I'd turn this into a glob to ease maintenance `(glob_files ../specs/*.json)`.

Reply:
Thanks! Done.